pyproject.toml

```
@@ -22,7 +22,10 @@ dependencies = [
     "torch >=1.10",
     "pytorch-lightning >=2, <3",
     "torchmetrics >1, <2",
```

```suggestion
    "torchmetrics[text] >=1.5, <2",
```
This will automatically catch nltk. Minimal version makes sure torchmetrics uses nltk's 'punkt_tab' instead of deprecated 'punkt'.